server/core/env.go

```diff
@@ -5,7 +5,7 @@ type (
 	EnvVariable struct {
 		ID           uint       `gorm:"primary_key;auto_increment;not null" json:"id"`
 		Key          string     `gorm:"not null" json:"key"`
-		Value        string     `gorm:"not null" sql:"type:text" json:"value"`
+		Value        string     `gorm:"not null" sql:"type:longtext" json:"value"`
```

Comment:
this only works for mysql unfortunately.

Reply:
Hm, I have no idea how to fix it for sqlite, except for truncating the actual value...

Reply:
As far as I tested, `longtext` works just fine in here with sqlite3 as backend, and if I got the docs correctly - sqlite doesn't limit the size.

Reply:
Tnx for the test, i will also test it and let you know